linera-service/src/server.rs

```diff
@@ -350,9 +350,9 @@ enum ServerCommand {
         #[arg(long, default_value = "10")]
         max_stream_queries: usize,
 
-        /// The maximal number of entries in the storage cache.
-        #[arg(long, default_value = "1000")]
-        cache_size: usize,
+        /// The storage cache policy
```

Comment:
This will generate a CLI w/o any helpful tips about what values are acceptable here. We should:
1. Improve the doc
2. and/or add a newtype wrapper for the `storage_cache_policy` with useful defaults.